test/workbox-background-sync/node/test-Queue.mjs

```diff
@@ -258,7 +260,7 @@ describe(`[workbox-background-sync] Queue`, function() {
       });
 
       const queue = new Queue('foo', {
-        maxRetentionTime: 1000,
+        maxRetentionTime: 1000 / MINUTES,
```

Comment:
Nit: I'm not sure what this is doing - 1000 / 60000 === 0.016666667 minutes? Could you just make this 1 minute and tick the clock by one minute and 1 ms?

Reply:
Ahh, yeah, for some reason I was thinking that we'd actually have to wait that long in the test.

Reply:
Done.